applications/StructuralMechanicsApplication/custom_utilities/tangent_operator_calculator_utility.h

```diff
@@ -327,17 +330,17 @@ class TangentOperatorCalculatorUtility
         double& rMaxValue
         )
     {
-        const std::size_t dimension = rArrayValues.size();
+        const SizeType dimension = rArrayValues.size();
         std::vector<double> non_zero_values;
 
-        for (std::size_t i = 1; i < dimension; ++i) {
+        for (IndexType i = 1; i < dimension; ++i) {
```

Comment:
very inefficient! to do the push back i mean

Reply:
True